index2.html

```diff
@@ -0,0 +1,149 @@
+      <ul class="languages">
+        <li><button class="language-button">html</button></li>
+        <li><button class="language-button">css</button></li>
+        <li><button class="language-button">javascript</button></li>
+      </ul>
+      <div><button class="see-project-button">See Project</button></div>
+    </div>
+  </div>
+  </section>
+
+  <section class="about-myself">
+    <header class="about-me-header">
+      <div><h2>About Myself</h2></div>
+    </header>
+    <div><p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    <div class="social-media">
+      <div class="headline"><a class="social-media-link" href="http://"> LET’S CONNECT</a></div>
+      <ul class="social-icons">
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon -GitHub.png" alt="github icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Linkedin.png" alt="linkedin icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Angelist.png" alt="angelist icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Twitter.png" alt="tweeter icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Medium.png" alt="medium icon"></a></li>
+      </ul>
+      </div>
+      <div><button class="get-my-resume-button">Get My Resume</button></div>
+      <ul class="self-profession">
+        <li class="language-items">
+        <div class="language-select">
+          <h3>Languages</h3>
+          <img class="dropdown" src="icon-images/Icon - Chevron - Down - Larger.png" alt="dropdown icon">
+        </div>
+          <div class="java-html-css-container">
+            <div class="language-type-container">
+              <img class="language-images" src="language-icons/javascript.png" alt="javascript icon">
+              <h5>javascript</h5>
+            </div>
+            <div  class="language-type-container">
+              <img class="language-images" src="language-icons/html.png" alt="html icon">
+              <h5>HTML</h5>
+            </div>
+            <div  class="language-type-container">
+              <img class="language-images" src="language-icons/css.png" alt="css icon">
+              <h5>css</h5>
+            </div>
+            
+          </div>
+        </li>
+        <li class="frameworks">
+          <h3>Frameworks</h3>
+          <img class="dropdown" src="icon-images/Icon - Chevron - Right - Larger.png" alt="right dropdowwm">
+        </li>
+        <li class="skills">
+          <h3>skills</h3>
+          <div><img class="dropdown" src="icon-images/Icon - Chevron - Right - Larger.png" alt="right dropdown"></div>
+        </li>
+      </ul>
+    </section>
+</body>
+</html>
```

Comment:
- [ ] The sections you created on this file should be created after the first section you created at the previous project(`index.html`). The sections should be added after the "section-header" so the user can scroll down and see this content when he/she loads the application. By creating this second `index2.html` the user can never access these sections you created now('work' and 'about') when he/she opens the `index.html`. Please, add these sections you created in this file to the `index.html` file and remember to add the style you created to that file too(whether to import the new `projects.css` or to copy the content of the `projects.css` to the `style.css`).